### PR TITLE
[11.0][IMP] purchase_request: Add Estimated cost field

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Additional information on line level that gives an estimated cost of products requested to purchase.

CC @Eficent

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by AdriaGForgeFlow — APPROVED** [review]
LGTM :+1: Functional and code review

File: purchase_request/views/purchase_request_view.xml [hunk 1]
```diff
@@ -92,6 +92,9 @@
                                 <field name="analytic_account_id"
                                        groups="analytic.group_analytic_accounting"/>
                                 <field name="date_required"/>
+                                <field name="estimated_cost" widget="monetary"/>
+                                <field name="currency_id" invisible="1"/>
+                                <field name="company_id" groups="base.group_multi_company" widget="selection"/>
```
Comment (nikul-serpentcs) [inline purchase_request/views/purchase_request_view.xml:97]: @hveficent remove `widget="selection"` and use `option={no_edit_create: true}`

File: purchase_request/views/purchase_request_view.xml [hunk 2]
```diff
@@ -277,6 +285,7 @@
                 <field name="analytic_account_id"
                        groups="analytic.group_analytic_accounting"/>
                 <field name="supplier_id"/>
+                <field name="company_id" groups="base.group_multi_company" widget="selection"/>
```
Comment (nikul-serpentcs) [inline purchase_request/views/purchase_request_view.xml:288]: same here

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Improve Code

File: purchase_request/views/purchase_request_view.xml (hunk 1 above)
Comment (HviorForgeFlow) [reply purchase_request/views/purchase_request_view.xml:97]: Which improvement we've got using this?

File: purchase_request/views/purchase_request_view.xml (hunk 2 above)
Comment (HviorForgeFlow) [reply purchase_request/views/purchase_request_view.xml:288]: same here

File: purchase_request/views/purchase_request_view.xml (hunk 1 above)
Comment (nikul-serpentcs) [reply purchase_request/views/purchase_request_view.xml:97]: Yes, If you using `widget="selection"` only see `80` records but if you `option={no_edit_create: true}`
then search more record.

Comment (HviorForgeFlow) [reply purchase_request/views/purchase_request_view.xml:97]: I think 80 records are fine for company related field. I'll keep it.

Comment (nikul-serpentcs) [reply purchase_request/views/purchase_request_view.xml:97]: @hveficent If you have possible then Change it.

**Comment by HviorForgeFlow** [comment]
@jbeficent it is ready to merge.

**Pull request merged**